components/supervisor-api/control.proto

```diff
@@ -44,6 +44,8 @@ message CreateSSHKeyPairResponse {
     string private_key = 1;
     // Return hostKey used by client to verify fingerprint
     SSHPublicKey host_key = 2;
+    // Return userName used by client to authenticate
+    string userName = 3;
```

Comment:
I think we actually should have user on workspace info, but I don't think it harms here, since otherwise we will need to add one more remote call to establish SSH connection.